tests/Application/src/Migrations/Version20171003103916.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+use Sylius\Component\Attribute\AttributeType\SelectAttributeType;
+use Sylius\Component\Product\Model\ProductAttributeInterface;
+use Symfony\Component\DependencyInjection\ContainerAwareInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+class Version20171003103916 extends AbstractMigration implements ContainerAwareInterface
```

Comment:
Is this the only possible way? I have bad memories with ContainerAware migrations